### PR TITLE
12 social networks login page

### LINKED ISSUE #12: Social networks login page

### PR DESCRIPTION
**Pull request by elnurastanov**

### CODE REVIEW

File: src/components/background/index.tsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
+import { Box, keyframes, useTheme } from "@mui/material";
+import { FC, PropsWithChildren } from "react";
+import LinkedInIcon from "@mui/icons-material/LinkedIn";
```
Comment (MDReal32) [inline src/components/background/index.tsx:3]: Likely import order.

packages form node_modules
absolute packages by root
relative packages
icons/csss

empty line between them and react must be first like core package

File: src/components/background/index.tsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+    100%{
+      transform: translateY(-1000px) rotate(720deg);
+      opacity: 0;
+      border-radius: 50%;
+    }
+  `;
+
+  return (
+    <Box
```
Comment (MDReal32) [inline src/components/background/index.tsx:23]: recomendable move to component and to scss file. (for easy writing)

File: src/components/icons/social.tsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
+import { SvgIcon } from "@mui/material";
+
+export const Social = ({ viewBox = "0 0 478.67 383.237", ...props }) => {
```
Comment (MDReal32) [inline src/components/icons/social.tsx:3]: type of props.

if same as SvhIcon then use like `FC<{ imported typeof SvgIcon props }>`

File: src/components/steps/customStepConnector.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import { StepConnector, stepConnectorClasses } from "@mui/material";
+import { styled } from "@mui/material/styles";
+
+export const CustomStepConnector = styled(StepConnector)(({ theme }) => ({
```
Comment (MDReal32) [inline src/components/steps/customStepConnector.ts:4]: I prefer separated scss file

File: src/constants/steps.ts [hunk 5]
```diff
@@ -0,0 +1,11 @@
+export const steps = [
+  {
+    label: "Choose your network",
+  },
+  {
+    label: "Select template ",
```
Comment (MDReal32) [inline src/constants/steps.ts:6]: space on end of string

File: src/components/steps/index.tsx [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+        top: theme.spacing(5),
+        left: "50%",
+        transform: "translateX(-50%)",
+        width: {
+          xs: "90%",
+          md: "70%"
+        }
+      }}
+    >
+      {Children.toArray(
```
Comment (MDReal32) [inline src/components/steps/index.tsx:27]: useless call of `Children.toArray`

**Review by MDReal32 — CHANGES_REQUESTED** [review]
.

File: src/components/steps/index.tsx (hunk 6 above)
Comment (elnurastanov) [reply src/components/steps/index.tsx:27]: Why? Children.toArray method prevents use key for each element and other performance issues

File: src/components/steps/customStepConnector.ts (hunk 4 above)
Comment (elnurastanov) [reply src/components/steps/customStepConnector.ts:4]: Separated scss file can’t provide theme property. This writing style recommended on material ui documentation

File: src/components/background/index.tsx (hunk 1 above)
Comment (elnurastanov) [reply src/components/background/index.tsx:3]: Auto import issue 😅

File: src/components/steps/index.tsx (hunk 6 above)
Comment (MDReal32) [reply src/components/steps/index.tsx:27]: okay

File: src/components/steps/customStepConnector.ts (hunk 4 above)
Comment (MDReal32) [reply src/components/steps/customStepConnector.ts:4]: okay. I get u.

**Review by MDReal32 — APPROVED** [review]

**Review by MDReal32 — APPROVED** [review]
There is no commits but ok.

**Pull request merged**